packages/daemon/src/rpc/methods/app.js

```diff
@@ -111,3 +133,32 @@ export const setPermission = async (
     await ctx.openVault.save()
   }
 }
+
+const constructClientSession = (
```

Comment:
Not an issue but could you name it `createClientSession` rather than `constructClientSession` to be more consistent with the rest of the codebase and other JS libraries please?